content/reference/drops/contact_request_drop.md

```diff
@@ -0,0 +1,25 @@
+# Contact Request Drop
+
+Allows to check and show Contact Form errors.
```

Comment:
IMHO we can remove *XXX Drop* from the start. Line above is header with same content. I think that is enough.